docs/articles/documentation/using-testcafe/programming-interface/runner.md

```diff
@@ -410,6 +411,41 @@ Parameter         | Type    | Description   Default
 runner.startApp('node server.js', 4000);
 ```
 
+### clientScripts
+
+Injects scripts into pages visited during the tests. Use this method to introduce client-side mock functions or helper scripts.
+
+```text
+async clientScripts( script[, script2[, ...[, scriptN]]] ) → this
+```
+
+Parameter | Type                | Description
+--------- | ------------------- | ------------
+`scripts` | String &#124; Array | Scripts to inject into the tested pages. See [Provide Scripts to Inject](common-concepts/inject-scripts-into-tested-pages.md#provide-scripts-to-inject) to learn how to specify them.
```

Comment:
I can't find `scripts` in the method signature `script[, script2[, ...[, scriptN]]] `